askup/templates/askup/question_answer_modal.html

```diff
@@ -1,66 +0,0 @@
-<div class="container-fluid">
```

Comment:
Why do you delete this template?

Reply:
It was transferred from the original Ruby site, but never used later, because of changes in the Quiz process.

Reply:
Got it.